### ISSUE TITLE
crashpad/20200508: cmake_find_package generator not working

### ISSUE THREAD

**Issue by EmilRosenquist** [bincrafters/community#1386]
### Package and Environment Details
  * Package Name/Version: **crashpad/20200508**
  * Operating System+version: **Windows 10 (Version	10.0.18363 Build 18363)**
  * Compiler+version: **Visual Studio 16**
  * Conan version: **conan 1.31.4**
  * Python version: **Python 3.9.0**

### Conan profile
```
Configuration for profile default:
[settings]
os=Windows
os_build=Windows
arch=x86_64
arch_build=x86_64
compiler=Visual Studio
compiler.version=16
build_type=Release
[options]
[build_requires]
[env]
```

### Steps to reproduce
1. Use `cmake_find_package` generator
	```
    [requires]
    crashpad/20200508@bincrafters/stable

    [generators]
    cmake_find_package
	```

2. Use `find_package` and `target_link_libraries` in CMakeLists.txt 
    ```
    ...
    find_package(crashpad REQUIRED)
    ...
    target_link_libraries(${PROJECT_NAME}
        PRIVATE
            crashpad::crashpad
    )
    ```
3. Include a file
    ```
    #include <client/settings.h>

    int main(int argc, char *argv[])
    {
	    return 0;
    }
    ```
4. Cant include the header.
    ```
    Error	C1083	Cannot open include file: 'client/settings.h': No such file or director
    ```

### Logs
<details><summary>Click to expand log</summary>

```
1> CMake generation started for configuration: 'x64-Release'.
1> The toolchain file has changed (CMAKE_TOOLCHAIN_FILE).
1> [CMake] -- The CXX compiler identification is MSVC 19.28.29336.0
1> [CMake] -- Detecting CXX compiler ABI info
1> [CMake] -- Detecting CXX compiler ABI info - done
1> [CMake] -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.28.29333/bin/Hostx64/x64/cl.exe - skipped
1> [CMake] -- Detecting CXX compile features
1> [CMake] -- Detecting CXX compile features - done
1> [CMake] 
1> [CMake] -- The C compiler identification is MSVC 19.28.29336.0
1> [CMake] -- Detecting C compiler ABI info
1> [CMake] -- Detecting C compiler ABI info - done
1> [CMake] -- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.28.29333/bin/Hostx64/x64/cl.exe - skipped
1> [CMake] -- Detecting C compile features
1> [CMake] -- Detecting C compile features - done
1> [CMake] -- Conan Setup - Compiler: Visual Studio
1> [CMake] -- Conan Setup - Compiler C: --env=CC=cl.exe
1> [CMake] -- Conan Setup - Compiler C++: --env=CXX=C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.28.29333/bin/Hostx64/x64/cl.exe
1> [CMake] -- Conan Setup - Compiler Version: --settings=compiler.version=16
1> [CMake] -- Conan Setup - Build Type: Release
1> [CMake] -- Conan Setup - Target Arch: x86_64
1> [CMake] -- Conan Setup - Host Arch: x86_64
1> [CMake] -- Conan Setup - Host OS: Windows
1> [CMake] -- Conan: Installing dependencies. Build missing=ON
1> [CMake] Configuration:
1> [CMake] [settings]
1> [CMake] arch=x86_64
1> [CMake] arch_build=x86_64
1> [CMake] build_type=Release
1> [CMake] compiler=Visual Studio
1> [CMake] compiler.runtime=MT
1> [CMake] compiler.version=16
1> [CMake] os=Windows
1> [CMake] os_build=Windows
1> [CMake] [options]
1> [CMake] [build_requires]
1> [CMake] [env]
1> [CMake] CC=cl.exe
1> [CMake] CXX=C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.28.29333/bin/Hostx64/x64/cl.exe
1> [CMake] conanfile.txt: Installing package
1> [CMake] Requirements
1> [CMake]     crashpad/20200508@bincrafters/stable from 'bincrafters' - Cache
1> [CMake] Packages
1> [CMake]     crashpad/20200508@bincrafters/stable:a79a557254fabcb77849dd623fed97c9c5ab7651 - Cache
1> [CMake] crashpad/20200508@bincrafters/stable: Already installed!
1> [CMake] conanfile.txt: Generator cmake_find_package created Findcrashpad.cmake
1> [CMake] -- Found crashpad: 20200508 (found version "20200508") 
1> [CMake] -- Configuring done
1> [CMake] -- Generating done
...


...
Then when building Building:

Severity	Code	Description	Project	File	Line	Suppression State
Error	C1083	Cannot open include file: 'client/settings.h': No such file or directory
```

</details>

**Comment by EmilRosenquist** [comment]
@Croydon Pinging you here since you were the one who merged #1381. Thanks!

**Comment by EmilRosenquist** [comment]
Nvm, User mistake!
